src/components/App/Header.css

```diff
@@ -3,7 +3,7 @@
 }
 
 .nc-appHeader-main {
-  @apply(--dropShadowMain);
+  @apply (--dropShadowMain);
```

Comment:
Even though the use of parentheses isn't mentioned in the [`@apply` spec](http://tabatkins.github.io/specs/css-apply-rule/#using), postcss-cssnext seems (at least in the [playground](http://cssnext.io/playground/)) to be able to transpile it, including with the additional space introduced in this change.

Incidentally, I believe the `@apply` feature [has been abandoned ](https://www.xanthir.com/b4o00). I can create a separate issue to discuss whether it should be refactored out if you like?

Reply:
I'm aware of the `@apply` stalling out, I'm expecting to just drop it whenever we switch to a CSS-in-JS solution.